### ISSUE TITLE
Change Directory Structure Read/Write

### ISSUE THREAD

**Issue by jmdelahanty** [Tyelab/bruker_control#57]
After discussion with @laurelrr on 11/8/21, the raw data directory has been updated to have an easier to understand/cleaner structure. Small changes in directory building in `bruker_control` have to be done to make sure that things are written to and read from the correct, updated locations.

- [x] Update directory builds in utility modules
- [x] Update directory choice searching in `main` file
- [x] Test with full session

**Comment by jmdelahanty** [comment]
Things have been reading/writing correctly since yesterday evening around 10PM (11/10/21) but, after updating Prairie View from 5.5.64.500 to 5.6.64.100, many things were broken.

Some bug reports were sent to Bruker and they're working on them now. As of today (11/11/21), Austin and I are rolling back to v.5.5.64.500 until further notice... This issue and the others relevant to it will remain open until 5.6 is stable and I can merge everything back to main.

**Comment by jmdelahanty** [comment]
Prairie View v.5.6.64.100 is not yet confirmed to be stable for the system. Austin has requested we remain on v5.5.64.500 until further notice/pending further testing and installation of hotfixes from Bruker which are coming soon. Steven from Bruker has suggested updating to 5.6 but I informed him I encountered problems that broke things. Deryn and I will be talking with him in the near future about several things and this will be one of them. Due to the desire to stay on the older version, I'll be closing this issue with the next pull request. A future PR will be initiated to get v5.6 onto main.
